TriangulationForm.html

```diff
@@ -149,19 +151,24 @@ <h4 class="panel-title">
 			TIPO DE VISITA<span class="required">*</span>
 			</label>
 			<select class="form-control input-section" id="obs.type_of_visit" required
-				name="obs.type_of_visit" data-concept="1981^TIPO DE ATENDIMENTO FAMILIAR^99DCT">
+				name="obs.type_of_visit" data-concept="1981^TIPO DE ATENDIMENTO FAMILIAR^99DCT" readonly="readonly" disabled >
 				<option value="">...</option>
 				<option value="2161^VISITA DE APOIO^99DCT">Visita preventiva</option>
 				<option value="2160^VISITA DE BUSCA^99DCT">Visita de Reintegração</option>
-				<option value="23914^VISITA AOS CASOS ESPECIAIS^99DCT">Visita aos casos especiais</option>
+				<option value="23914^VISITA AOS CASOS ESPECIAIS/CM^99DCT">Clínica Móvel</option>
+				<option value="23914^VISITA AOS CASOS ESPECIAIS/AMA^99DCT">Aconselhamento para Melhoria de Adesão (AMA)</option>
+				<option value="2160^VISITA DE BUSCA/DCP^99DCT">Dispensa Comunitária via Provedor</option>
 			</select>
+			<input class="form-control" name="obs_comment.type_of_visit"
+                   data-obscommentfor="obs.type_of_visit"  id="obs_comment.type_of_visit"
+                    value="AMA" type="hidden">
```

Comment:
Remove the value AMA since the value is supposed to be filled via javascript, then resolve the conflicts